src/groupeddataframe/splitapplycombine.jl

```diff
@@ -448,31 +486,38 @@ combine(f::Union{Base.Callable, Pair}, gd::GroupedDataFrame; keepkeys::Bool=true
     combine(gd, f, keepkeys=keepkeys)
 combine(gd::GroupedDataFrame, f::Base.Callable; keepkeys::Bool=true) =
     combine_helper(f, gd, keepkeys=keepkeys)
+combine(gd::GroupedDataFrame, f::typeof(nrow); keepkeys::Bool=true) =
+    combine(gd, [nrow => :nrow], keepkeys=keepkeys)
 
 function combine(gd::GroupedDataFrame, f::Pair; keepkeys::Bool=true)
     # move handling of aggregate to specialized combine
-    if isagg(first(f) => (last(f) isa Pair ? first(last(f)) : last(f)))
+    f_from, f_to = f
+
+    # verify if it is not better to use a fast path
```

Comment:
So I understand the fast path is implemented in the `combine(::GroupedDataFrame, ::AbstractArray)` method? Maybe mention it in the comment.

Reply:
It is implemented in `_combine(::AbstractVector, ::GroupedDataFrame)` (so that we can also redirect `map` there). But here we redirect to `combine` because `combine` does some additional preprocessing and we want to be consistent.

I will add a comment.